### PR TITLE
Lab 3 Complete

### PR DESCRIPTION
**Pull request by Denchidlo**
Oh shishh.. Here we go again. 

Here I've prepared some frontend stuff for you (HTML and CSS). Don't be thrown by the commit order here since I've produced deploy branch (smth like dev version) and all future commits would have its origin

### CODE REVIEW

**Comment by Denchidlo** [comment]
Btw see deploy branch to observe results of all of my struggles and sufferings)

File: studio/projects.html [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        <meta name="description" content="">
+        <meta name="author" content="Dzianis Ivanchuk">
+        <link rel="icon" href="./../src/static/logo.png">
+        <title>Annotate.ai - Studio</title>
+        <link href="./../src/css/pages/projects.css" rel="stylesheet">
+    </head>
+
+    <body class="project-body">
+        <header class="menu-header">
+            <div class="menu-header__trigger">
```
Comment (katetushkan) [inline studio/projects.html:16]: `section` would be better here.

File: studio/projects.html [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+                <div class="menu-logo">
+                    <img src="./../src/static/logo.png" height="24px">
+                    <label>AI Studio</label>
```
Comment (katetushkan) [inline studio/projects.html:19]: `label` can't live without `input`, so go with `<p>`, '<span>', '<strong>' would be correct.

File: studio/projects.html [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+                                <input type="checkbox" id="user-action-list" class="invisible">
+                                <img class="user-picture__avatar">
+                                <span class="user-picture__username">DI</span>
+                            </div>
+                        </div>
+                    </a>
+                </li>
+                <li class="projects-page__card">
+                    <a class="projects-page__card-link" href="./project.html">
+                        <div class="projects-page__card-header">
```
Comment (katetushkan) [inline studio/projects.html:66]: Maybe, if that's a `-header` , so use `<header>`?)

File: studio/projects.html [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+                            <p class="projects-page__card-title">Animal Classification #2</p>
+                            <section class="projects-page__card-details">
+                                <p class="projects-page__card-details__task-type">Task: Image segmentation</p>
+                                <p class="projects-page__card-details__completion-stat">Ready: 10 / 100</p>
+                            </section>
+                        </div>
```
Comment (katetushkan) [inline studio/projects.html:72]: u can make It `article`.

File: studio/collaborators.html [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+    </header>
+
+    <section class="overlay-form-user-info__main-content">
+        <form class="overlay-form-user-info__main-content-user-form">
+            <input id="overlay-form-user-info__main-content-user-name">
+            <input id="overlay-form-user-info__main-content-user-email">
+            <input id="overlay-form-user-info__main-content-user-short">
+        </form>
+        <section class="overlay-form-user-info__main-content-stat">
+            <div class="overlay-form-user-info__main-content-stat-tasks-completed">
```
Comment (katetushkan) [inline studio/collaborators.html:194]: it's better use `<dl>` [tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) for this

File: src/css/forms/common.css [hunk 6]
```diff
@@ -0,0 +1,106 @@
+
+/* Page styles */
+  
+/* Overlay Styles */
+.overlayBg {
```
Comment (katetushkan) [inline src/css/forms/common.css:5]: couldn't find this class in html...(

File: src/css/forms/common.css [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+  position: fixed;
+  left: 0;
+  top: 0;
+  z-index: 5;
+  width: 100vw;
```
Comment (katetushkan) [inline src/css/forms/common.css:10]: if u wanna stretch ur element full screen better use % for this, viewport values can be tricky on mobile devices.

**Review by katetushkan — CHANGES_REQUESTED** [review]

File: src/css/forms/common.css (hunk 6 above)
Comment (Denchidlo) [reply src/css/forms/common.css:5]: Fixed it, u faced it as this class was automatically created during overlay init

File: src/css/forms/common.css (hunk 7 above)
Comment (Denchidlo) [reply src/css/forms/common.css]: Fixed it, thx a lot!

**Review by katetushkan — APPROVED** [review]
Good. That's <kbd>9</kbd>.

**Pull request merged**